src/containers/LearningResourcePage/CreateLearningResource.jsx

```diff
@@ -38,7 +38,7 @@ class CreateLearningResource extends Component {
     const { locale, ...rest } = this.props;
     return (
       <LearningResourceForm
-        history={history}
+        history={history} /* eslint-disable-line */
```

Comment:
Er skeptisk til å disable eslint overalt. Trengs denne?

Reply:
`history` og `location` har havnet på lista til `no-restricted-globals` (https://eslint.org/docs/rules/no-restricted-globals). P.t går det ikke å sette tillatte globals for regelen. Workaround som løser dette er å referere direkte til prop-en via `this.prop...`